desktop/containerd/index.md

```diff
@@ -209,7 +209,7 @@ View Tags on DockerHub to see multi-platform result:
 
 ## Containerd Image Store Release Notes
 
-(2022-10-19)  
+### 2022-10-19 
 New commands and multiple bug fixes included with Docker Desktop 4.13.0.
 
 ### New
```

Comment:
The New and Known issues section appear at the same level the release date. Should this be H4?